iotc-azrtos-sdk/src/iotconnect_di.c

```diff
@@ -0,0 +1,539 @@
+//
+// Copyright: Avnet, Softweb Inc. 2022
+// Created by Nik Markovic <nikola.markovic@avnet.com> on 5/15/22.
+//
+
+#include <stdlib.h>
+#include <stdint.h>
+#include <stdbool.h>
+#include <ctype.h>
+//TODO: remove remaining dependencies to TO
+#include "TO.h"
```

Comment:
TODO?? Not sure where TO.h comes from?

Reply:
TO.h comes from TO lib (pulled by setup-project.sh). We are using it for some size declarations. I took a look at what can be done to get rid of the TO dependency and I found a bug in @lagosantol 's code. NX_CRYPTO_HASH_SHA256 is over 30k hex in size. It seems that a wrong define is used. @lagosantol can you test? 

The problem is that we are currently not set up to test this code.

Reply:
TO lib (Trusted Objects lib for TO136 chip) comes from setupp-project.sh.

I will have to put this off (this = removing the dependency) for later because I don't have a DDIM setup that I can use to test the change.